doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc

```diff
@@ -0,0 +1,21 @@
+[id='create_archetype_project']
+= Creating a project using archetypes
+
+Archetypes are projects that are installed in Apache Maven repositories and contain a specific template structure. You can also generate parameterized versions of the project templates using archetypes. You can use archetypes to create a project in {CENTRAL}. When you create a project, it is added to the Git repository that is connected to your {PRODUCT} installation.
+
+.Prerequisites
+
+* You have created an archetype and listed in the {CENTRAL} *Settings*.
+
+* You have set a default archetype in your space in {CENTRAL}.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects*.
+. Click *Add Project*.
+. Type the project name and description in the *Name* and *Description* fields respectively.
+. Click *Configure Advanced Options*.
+. Select the *Based on template* checkbox.
+. If you want to use an archetype other than the default archetype, select an archetype from the drop-down options.
```

Comment:
Not sure about 'drop-down options'. Do you click the existing archetype name to see a list of other archetypes? If the  drop-down doesn't have a name you need to describe it something like I have.

Reply:
how about changing to this "If you want to use an archetype other than default archetype, click on the default archetype and select an archetype from the drop-down."